### PR TITLE
HPCC-27862 Add Logging Analytics details to Doc

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>
Note: Composed PDF File can be viewed: 
http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/lastSuccessfulBuild/artifact/build-docs/Release/docs/EN_US/Containerized_HPCC_Systems_Platform_EN_US-8.8.0-closedown0.pdf#page=52

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful Unit Testing completed on Region 8 Jenkins system: http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/

### CODE REVIEW

**Comment by rpastrana** [comment]
@g-pan  looks good. this sentence isn't 100% clear to me, specially because it doesn't provide the script name, but also because it's not clear that the user should populate the values in the accompanying env-loganalytics file.

The script located at  https://github.com/hpcc-systems/HPCC-Platform/tree/master/helm/examples/azure/log-analyt-ics requires populating a few values in the env-loganalytics environment file.

**Review by rpastrana — APPROVED** [review]
It's difficult to discern the code review change since it was squashed, but it seems fine.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 1]
```diff
@@ -257,49 +255,183 @@ myelk-kibana LoadBalancer 10.110.129.199 localhost 5601:31465/TCP 68m
...
+    should be well understood before implementation. Log content is written to
+    the logs store associated with your Log Analytics workspace.</para>
 
     <sect2>
       <title>Enabling Azure Log Analytics</title>
 
-      <para>Enabling Azure's Log Analytics on the target AKS cluster can be
-      done from the Azure portal or via CLI. For detailed Azure documentation:
-      Enable Container Log Analytics:</para>
+      <para>Enabling Azure's Log Analytics (ALA) on the target AKS cluster can
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:342]: Enable Azure's Log Analytics (ALA) on the target AKS cluster using one of the following options:

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 2]
```diff
@@ -257,49 +255,183 @@ myelk-kibana LoadBalancer 10.110.129.199 localhost 5601:31465/TCP 68m
...
+        <para>Provide these values in the
+        <emphasis>env-loganalytics</emphasis> environment file order to create
+        a new Azure Log Analytics workspace, associate it with a target AKS
+        cluster, and enable the processing of logs:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para><emphasis role="bold">LOGANALYTICS_WORKSPACE_NAME</emphasis>
+            The desired name for the Azure LogAnalytics workspace to be
+            associated with target AKS cluster. A new workspace will be
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:398]: A new workspace is created if this value does not exist

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Couple of minor comments inline

**Review by JamesDeFabia — APPROVED** [review]
Good from my POV

**Comment by g-pan** [comment]
good to merge

**Pull request merged**